nvme-print.c

```diff
@@ -763,10 +763,19 @@ void nvme_show_endurance_log(struct nvme_endurance_group_log *endurance_log,
 	nvme_print(endurance_log, flags, endurance_log, group_id, devname);
 }
 
+const char *nvme_degrees_string(long t, bool fahrenheit)
+{
+	static char str[STR_LEN];
+
+	sprintf(str, "%ld °%s", kelvin_to_degrees(t, fahrenheit), fahrenheit ? "F" : "C");
```

Comment:
I think it would better to call the right kelvin_to_TYPE directly and get rid of kelvin_to_degrees, e.g.

```c

long val;

if (fahrenheit) 
  val = kelvin_to_fahrenheit(t);
else
  val = kelvin_to_celsius(t);

sprintf(str, "%ld °%s", val, fahrenheit ? "F" : "C");
```

Reply:
Fixed as mentioned.